src/Redux/Categories/Categories.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const CHECK = 'bookstore/categories/CHECK';
+
+const initialState = {
+  categories: [],
+};
+
+const categories = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkCategories(state, action) {
+      if (action.payload === CHECK) {
+        // console.log('Check Status');
```

Comment:
- **Please remove this commented code to make your work cleaner.**
- **Make sure the check status action returns a string** "Under construction"